eng/scripts/Language-Settings.ps1

```diff
@@ -744,9 +746,10 @@ function Validate-java-DocMsPackages ($PackageInfo, $PackageInfos, $DocValidatio
 
   if (!(ValidatePackages $PackageInfos $DocValidationImageId)) {
```

Comment:
Is this validation going to fail fast or will it keep going if one of them is bad?

Reply:
This validation keeps going if one of the validations fails because `Update-DocsMsMetadata.ps1` calls `Validate-java-DocMsPackages` once for each package so the end result is as expected.

`FallbackValidation` collects results together and checks whether there's a `$false`  https://github.com/Azure/azure-sdk-for-java/blob/ae08c9f883e195fd14741a48dbdd4a7851fa1c3e/eng/scripts/Language-Settings.ps1#L416C19-L416C19

`DockerValidation` will fail fast if one of the packages fails validation.